content/attacks/posts/2021-02-13-Alpha-Finance.md

```diff
@@ -0,0 +1,47 @@
+---
+date: 2021-02-13
+categories:
+    - 51%

```

Comment:
It's not a 51% attack
> At the end of the day, the Alpha Homora attacker was able to make approximately $37.5 million by exploiting the vulnerable contract.

I recommend to investigate the case, trying to understand it before writing an article. 
So category is DeFi and Flash loan attack